### PR TITLE
Add project details popup feature

### PR DESCRIPTION
**Pull request by ahmidou09**
**This pull request adds the functionality to display project details in a popup window.**

- The popup appears when the user clicks (or taps) on the 'Check Details' button for each project.

- The popup contains information such as the project name, description, featured image, technologies used, and links to the live version and source code. Additionally, the close button allows the user to dismiss the popup when they're done reviewing the project details. 
- This implementation utilizes a JavaScript array to store information for all projects, allowing for dynamic creation of the projects section on the main page. 

- The popup window is designed to be responsive and works seamlessly on both mobile and desktop screen sizes.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -3,6 +3,43 @@ const toggelBtn = document.querySelector('.toggel-btn');
...
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy  text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+  tags: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+},
+{
+  card: 4,
+  title: 'Uber Navigation',
+  items: ['Facebook', 'Back End Dev', '2011'],
+  image: './img/Snapshoot-Portfolio-popup.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy  text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+  tags: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
```
Comment (Ismailco) [inline main.js:41]: - [ ] Great work on the pop-up, however, the `data` (projects array) should include a `live demo link` and a `code source link`, kindly consider adding these links to the objects in the `data` array

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ahmidou09, @jcmunav63,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @ahmidou09, @jcmunav63, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmidou09** [comment]
Ok, thanks for your valuable comment

**Pull request merged**